docker/docker-compose.test.yml

```diff
@@ -0,0 +1,11 @@
+version: "3"
+services:
+  notification-send:
+    build:
+      context: ..
+      dockerfile: docker/Dockerfile.rails
+    volumes:
+      - ..:/app
+    command: "rake notifications:send"
+    environment:
+      KAFKA_BROKER_HOST: kafka
```

Comment:
I am not sure how it works, but I still think we should have `depends_on` section with `racecar` and `resque`